torch/_subclasses/fake_tensor.py

```diff
@@ -541,6 +557,8 @@ def __new__(cls, fake_mode, elem, device, constant=None, real_tensor=None):
         self.real_tensor = real_tensor  # type: ignore[attr-defined]
         self._nonzero_memo = None  # type: ignore[attr-defined]
```

Comment:
Worth generalizing this ? @IvanKobzarev was also recently asking for `.tolist()` memoization

Reply:
alright